test/integration_test.py

```diff
@@ -639,26 +642,25 @@ def run(thread_count):
                         }
                     ],
                     "mock-count": entry_format.format(thread_count, op_count)
-                })
-                portal_service._crud(lambda db: list(db) + [mock_entry])
-
-        old_db = portal_service.read()
+                }
+                self.portal_service._crud(lambda db: [*db, mock_entry])
```

Comment:
please move all cosmetic changes to a separate commit.

Reply:
https://github.com/DataBiosphere/azul/pull/2602/commits/9ac3ab01c9d3332ccba9fcfea5f64f19dda4107c